GSA/digitalgov.gov
# Add files via upload

Adding icon and logo for AI.gov, which we need for the "Cloud and infrastructure" topic page

## Summary

I am adding to https://github.com/GSA/digitalgov.gov/tree/main/static/logos the icon and logo for AI.gov.




### Solution

### How To Test

1. Navigate to https://github.com/GSA/digitalgov.gov/tree/main/static/logos
2. Make sure that icon (50 px x 50px) and logo (250 px x 250 px) for AI.gov have been uploaded and are available for use


---

### Dev Checklist
